css/trackers.scss

```diff
@@ -98,6 +98,16 @@
             left: 7px;
         }
 
+        .ddg-logo-moz {
+            background-image: url($ddg-logo__url-moz);
+            background-size: cover;
+            width: 25px;
+            height: 25px;
+            position: absolute;
+            top: 7px;
+            left: 7px;
+        }
+
```

Comment:
...then here, just leave all the rules except the `background-image` in a generic `.ddg-logo` rule. Then within `.ddg-logo { .... }` rules, nest each class, both `.ddg-logo--moz` and `.ddg-logo--chrome` (<-- note two dashes) with the background-image url for each.